pinot-core/src/main/java/com/linkedin/pinot/core/operator/aggregation/AggregationOperator.java

```diff
@@ -46,23 +45,23 @@
    * Constructor for the class.
    *
    * @param aggregationsInfoList List of AggregationInfo (contains context for applying aggregation functions).
+   * @param executor Aggregation Executor
    * @param projectionOperator Projection operator.
    * @param numTotalRawDocs Number of total raw documents.
    */
-  public AggregationOperator(List<AggregationInfo> aggregationsInfoList, MProjectionOperator projectionOperator,
-      long numTotalRawDocs) {
+  public AggregationOperator(List<AggregationInfo> aggregationsInfoList, AggregationExecutor executor,
+      MProjectionOperator projectionOperator,long numTotalRawDocs) {
```

Comment:
Missing what space after 'projectionOperator,' -> Can you format the file as per the styling?